FirebaseScripts/CredentialsHelper.py

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
Instead of doing this manually, pull the config from a .env file. In the `README.md`, you can specify the need of `.env` file to run this script.

Reply:
nice suggestion will do the same

Reply:
hey, @antrikshmisri I tried using .env but I do not think that should be used as this is a script basically for beginners and I have tried it they would have to do a bit of formating as we need to send the credentials in a form of dict to firebase API. this may be confusing for them if you think that's ok I will do the same but I personally don't think we should do the same.

Reply:
@shantamsultania This is a common practice to store sensitive data in `.env`, I would suggest you to create the environment variables with empty strings, import those into your script and do mention the necessity for the user to add firebase credentials in the `.env` file in `README.md`.